imitation/util/util.py

```diff
@@ -11,16 +11,16 @@
 
 
 def maybe_load_env(env_or_str, vectorize=True):
-  """
-  Params:
-  env_or_str (str or gym.Env): The Env or its string id in Gym.
-  vectorize (bool): If True, then vectorize the environment before returning,
-    if it isn't already vectorized.
+  """Maybe load and process an environment or environment id.
```

Comment:
Can we explain what the "maybe" is doing here? When would or wouldn't this happen?